app/main.py

```diff
@@ -1,6 +1,25 @@
+import requests
+import os
+import json
+
+URL = "https://api.weatherapi.com/v1/current.json?"
+FILTERING = "Paris"
+KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    result_from_weather_api = requests.get(URL + f"q={FILTERING}&key={KEY}")
+
+    weather_dict = json.loads(result_from_weather_api.text)
```

Comment:
It would be better to use request object .json() method
```suggestion
    weather_dict =result_from_weather_api.json()
```